new-site/index.html

```diff
@@ -1,32 +1,162 @@
+        alt="Angular"
+      />
+      <h2>Angular</h2>
+      <p>Some points about Angular</p>
+      <ul>
+        <li>HTTP client</li>
+        <li>Services</li>
+        <li>Cross platform</li>
+        <li>Expression</li>
+      </ul>
+      <button type="submit" class="button">
```

Comment:
Submit buttons are for forms, but this is not a form. You can probably just use an `a` tag and style it to look like a button.